Source/WebKit/WPEPlatform/wpe/drm/WPEDRMSessionLogind.cpp

```diff
@@ -28,6 +28,7 @@
 
 #if ENABLE(JOURNALD_LOG)
 #include <errno.h>
+#include <glib-unix.h>
```

Comment:
The [documentation](https://docs.gtk.org/gio/class.UnixFDList.html) says that the header to use is `<gio/gunixfdlist.h>`, but only _before_ GLib 2.74; later versions support this functionality in the general library (there is a Windows implementation now) and that's what we did not notice this in the bots.

Ideally we would like to use imported targets for GLib, which would be less error prone, but nobody has gotten round to that (yet).